Ryujinx/Ui/MainWindow.cs

```diff
@@ -827,11 +830,13 @@ private void Row_Activated(object sender, RowActivatedArgs args)
         private void VSyncStatus_Clicked(object sender, ButtonReleaseEventArgs args)
         {
             _emulationContext.EnableDeviceVsync = !_emulationContext.EnableDeviceVsync;
+            Logger.Notice.Print(LogClass.Application, $"VSync changed to: {_emulationContext.EnableDeviceVsync}");
```

Comment:
```suggestion
            Logger.Notice.Print(LogClass.Application, $"VSync toggled to: {_emulationContext.EnableDeviceVsync}");
```
and add a new empty line before